src/WPBT/WPBT_Beta_RC.php

```diff
@@ -411,18 +411,25 @@ public function add_dashboard_widget() {
 	public function beta_tester_dashboard() {
 		$next_version       = $this->next_package_versions();
 		$milestone          = array_shift( $next_version );
-		$wpbt_settings_page = add_query_arg( 'page', 'wp_beta_tester', admin_url( 'tools.php' ) );
 
 		/* translators: %s: WordPress version */
 		printf( wp_kses_post( __( 'Please help test <strong>WordPress %s</strong>.', 'wordpress-beta-tester' ) ), esc_attr( $milestone ) );
 
 		echo wp_kses_post( $this->parse_development_feed( $milestone ) );
 
-		/* translators: %1: link to closed and reopened trac tickets on current milestone */
-		printf( wp_kses_post( '<p>' . __( 'Here are the <a href="%s">commits for the milestone</a>.', 'wordpress-beta-tester' ) . '</p>' ), esc_url_raw( "https://core.trac.wordpress.org/query?status=closed&status=reopened&milestone=$milestone" ) );
+		/* translators: %1: link to closed trac tickets on current milestone */
+		printf( wp_kses_post( '<p>' . __( 'Here are the <a href="%s">commits for the milestone</a>.', 'wordpress-beta-tester' ) . '</p>' ), esc_url_raw( "https://core.trac.wordpress.org/query?status=closed&milestone=$milestone" ) );
```

Comment:
I think I changed this before you made your commit.
```suggestion
		printf( wp_kses_post( '<p>' . __( 'Here are the <a href="%s">commits for the milestone</a>.', 'wordpress-beta-tester' ) . '</p>' ), esc_url_raw( "https://core.trac.wordpress.org/query?status=closed&status=reopened&milestone=$milestone" ) );
```